tests/unit/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ foreach(test_name tests_high_five_base tests_high_five_multi_dims tests_high_fiv
   add_executable(${test_name} "${test_name}.cpp")
   target_link_libraries(${test_name}
                         ${Boost_UNIT_TEST_FRAMEWORK_LIBRARIES}
-                        ${HDF5_C_LIBRARIES})
+                        ${HDF5_C_LIBRARIES} Boost::serialization)
   if(HIGHFIVE_PARALLEL_HDF5)
     target_link_libraries(${test_name} ${MPI_C_LIBRARIES})
```

Comment:
Just for my own education, where is this required? On RH, the MPI wrapper takes care of setting up the correct include and link flags.

Reply:
https://travis-ci.org/BlueBrain/HighFive/builds/596668670#L842
I don't dig it.

Reply:
Maybe I can remote it
i will git it a try

Reply:
In fact, highfive got a dependency on serialization.
https://github.com/BlueBrain/HighFive/blob/master/include/highfive/H5DataSpace.hpp#L21

Reply:
Sorry, was commenting on the `MPI_C_LIBRARIES` actually.

Reply:
Thank you @matz-e 
This should be handled in a separate issue #219